Makefile

```diff
@@ -205,6 +205,11 @@ web: $(WEB_MARKDOWN_TARGETS) $(WEB_IMAGE_TARGETS) $(WEB_STATIC_TARGETS) $(READIN
 web_zip: web ## Create website and archive
 	cd $(WEB_OUTPUT_DIR) && rm -rf site.zip && zip -r site.zip *
 
+## TODO: temp, should remove
```

Comment:
Der Kommentar deutet darauf hin, dass dieses Target eigentlich wieder raus sollte? Und ich bin mir fast sicher, dass die Markdown-Dateien bereits in einer anderen Regel in eine Variable zusammengesucht werden?

Reply:
Ist raus